src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts

```diff
@@ -120,225 +191,280 @@ export class ExportPdfDialog extends BaseDialog {
-					titlePadding = (15 / 2) * (maxTitleRows - title.length);
-					yPos += titlePadding;
-				}
-				for (let i = 0; i < maxTitleRows; i = i + 1) {
-					if (title[i]) {
-						yPos += 15;
-						this.doc.text(title[i], pageWidth / 2, yPos, {
+				html2canvas(this.outputElement.nativeElement, {
+					backgroundColor: null,
+					scale: scaleFactor,
+				})
+					.then((canvas) => {
+						canvas.height = 100 * scaleFactor;
+						canvas.width = 100 * scaleFactor;
+						const image = this.generatePdfBackground(canvas);
+
+						const logoWidth = 50;
+						const logoHeight = 30;
+						const marginXImageLogo = 5;
+
+						let firstName = this.profile.firstName;
+						let lastName = this.profile.lastName;
+						firstName = firstName.charAt(0).toUpperCase() + firstName.slice(1);
+						lastName = lastName.charAt(0).toUpperCase() + lastName.slice(1);
+
+						// calculate competencies firs to know how many pages the PDF will have
+						// and to apply the background on each page
+						if (num_competencies > 0) {
+							const esco = competencies.some((c) => c.escoID);
+							const competenciesPerPage = 10;
+							this.doc.addPage();
+							this.doc.addImage(image, 'PNG', 0, 0, pageWidth, pageHeight, undefined, 'NONE');
+
+							//  OEB Logo
+							this.doc.addImage(oeb_logo, 'PNG', marginXImageLogo, yPos + 10, logoWidth, logoHeight);
+							this.doc.setDrawColor('#492E98');
+							this.doc.line(pageWidth / 2 - 50, 25, pageWidth / 2 + 100, 25);
+
+							if (esco) {
+								this.doc.textWithLink(
+									'* Kompetenz nach ESCO: https://esco.ec.europa.eu/de',
+									10,
+									pageHeight - 20,
+									{
+										url: 'https://esco.ec.europa.eu/de',
+									},
+								);
+							}
+
+							for (let i = 0; i < num_competencies; i++) {
+								if (i != 0 && i % competenciesPerPage === 0) {
+									this.doc.addPage();
+									this.doc.addImage(image, 'PNG', 0, 0, pageWidth, pageHeight, undefined, 'NONE');
+									//  OEB Logo
+									this.doc.addImage(
+										oeb_logo,
+										'PNG',
+										marginXImageLogo,
+										yPos + 10,
+										logoWidth,
+										logoHeight,
+									);
+									this.doc.setDrawColor('#492E98');
+									this.doc.line(pageWidth / 2 - 50, 25, pageWidth / 2 + 100, 25);
+								}
+
+								this.doc.setFontSize(28);
+								this.doc.setFont('Helvetica', 'bold');
+								this.doc.setTextColor('#492E98');
+								this.doc.text('Kompetenzen', 10, 60);
+
+								this.doc.setFontSize(18);
+								this.doc.setFont('Helvetica', 'normal');
+								this.doc.text('die ', 10, 75);
+								this.doc.setFont('Helvetica', 'bold');
+								this.doc.text(firstName + ' ' + lastName, this.doc.getTextWidth('die ') + 10, 75);
+								this.doc.setFont('Helvetica', 'normal');
+								const textWidth =
+									this.doc.getTextWidth('die ') +
+									15 +
+									this.doc.getTextWidth(firstName + ' ' + lastName);
+								this.doc.text('mit dem Badge', textWidth, 75);
```

Comment:
Would it make sense to have the text not hardcoded here? I imagine this could be an easy spot to miss if the text of the PDF was ever to be changed

Reply:
Maybe we can redo this when we add translations?

Reply:
Ok, but then maybe a TODO comment would be good?

Reply:
you mean in this specific place or everywhere where text is hardcoded? But maybe it is easier to spot now, that every section is in a function

Reply:
I meant in this specific place, so that we don't forget it

Reply:
added it